src/components/app/accountsState.ts

```diff
@@ -0,0 +1,80 @@
+// eslint-disable-next-line max-classes-per-file
```

Comment:
Note- We need to note if this file coupled to the UI, or it can define a general way to handle and track events, and if so, it should be in the "backend" side.

Reply:
Good question. I assumed this is ui state (I was planning to move it to the store) and thats why I put it under the ui code. If we see need for such state in the backend code, either will move this there or create another state there with just what the backend needs.